rasa/cli/arguments/train.py

```diff
@@ -20,52 +23,63 @@ def set_train_arguments(parser: argparse.ArgumentParser):
     add_augmentation_param(parser)
     add_debug_plots_param(parser)
 
-    add_num_threads_param(parser)
+    _add_num_threads_param(parser)
 
-    add_model_name_param(parser)
+    _add_model_name_param(parser)
     add_persist_nlu_data_param(parser)
     add_force_param(parser)
+    add_finetune_params(parser)
 
 
-def set_train_core_arguments(parser: argparse.ArgumentParser):
+def set_train_core_arguments(parser: argparse.ArgumentParser) -> None:
+    """Specifies CLI arguments for `rasa train core`."""
     add_stories_param(parser)
     add_domain_param(parser)
-    add_core_config_param(parser)
+    _add_core_config_param(parser)
     add_out_param(parser, help_text="Directory where your models should be stored.")
 
     add_augmentation_param(parser)
     add_debug_plots_param(parser)
 
     add_force_param(parser)
 
-    add_model_name_param(parser)
+    _add_model_name_param(parser)
 
     compare_arguments = parser.add_argument_group("Comparison Arguments")
-    add_compare_params(compare_arguments)
+    _add_compare_params(compare_arguments)
+    add_finetune_params(parser)
 
 
-def set_train_nlu_arguments(parser: argparse.ArgumentParser):
+def set_train_nlu_arguments(parser: argparse.ArgumentParser) -> None:
+    """Specifies CLI arguments for `rasa train nlu`."""
     add_config_param(parser)
     add_domain_param(parser, default=None)
     add_out_param(parser, help_text="Directory where your models should be stored.")
 
     add_nlu_data_param(parser, help_text="File or folder containing your NLU data.")
 
-    add_num_threads_param(parser)
+    _add_num_threads_param(parser)
 
-    add_model_name_param(parser)
+    _add_model_name_param(parser)
     add_persist_nlu_data_param(parser)
+    add_finetune_params(parser)
```

Comment:
@wochinge Just so I understand correctly - This line ensures that `rasa train nlu finetune` will also work in addition to `rasa train finetune`(currently mentioned in the PR), right?

Reply:
yep, I covered all three cases (`rasa train`, `rasa train core`, `rasa train nlu`)